src/jit/assertionprop.cpp

```diff
@@ -320,17 +321,17 @@ void Compiler::optAddCopies()
 #endif
 
             /* We have already calculated paramImportantUseDom above. */
-
-            BLOCKSET_ITER_INIT(this, iter, paramImportantUseDom, blkNum);
-            while (iter.NextElem(&blkNum))
+            BlockSetOps::Iter iter(this, paramImportantUseDom);
+            unsigned          bbNum = 0;
+            while (iter.NextElem(&bbNum))
```

Comment:
IMO, "hiding" a variable declaration in a `for` doesn't provide any benefit, but I don't care much one way or the other.

As for not initializing it, that's fine as long as the compilers don't give an uninitialized use warning.